Remove declaration of direct Wayland support

bk138 opened the pull request:
Similar what was experienced in #11, the main game session fails to start on Wayland (main menu works). One workaround is to select `osggraph` as the graphics engine, but I reckon there's a reason the other `ssggraph` is selected as default?

Anyway, to make the game start correctly and with ssggraph out of the box, _direct_ wayland support has to be disabled until the underlying issue w/ `ssggraph` and Wayland is fixed.

AsciiWolf commented:
/cc @son-link @hfiguiere

hfiguiere commented on org.speed_dreams.SpeedDreams.yaml:
```diff
@@ -5,7 +5,6 @@ sdk: org.freedesktop.Sdk
 command: speed-dreams-2
 
 finish-args:
-  - --socket=wayland
```
you can't remove this and keep `fallback-x11`. it must be replaced by `x11`

bk138 replied:
Updated the commit.

bbhtt commented:
bot, build

bk138 committed:
Remove declaration of direct Wayland support

Similar what was experienced in #11, the main game session fails to start
on Wayland (main menu works). One workaround is to select `osggraph` as
the graphics engine, but I reckon there's a reason the other `ssggraph`
is selected as default?

Anyway, to make the game start correctly and with ssggraph out of the box,
_direct_ wayland support has to be disabled until the underlying issue
w/ `ssggraph` and Wayland is fixed.

hfiguiere commented:
bot, build

hfiguiere commented:
looks like tarball download problems.

bot, build

bk138 committed:
Merge branch 'master' into master
